include/openssl/tls1.h

```diff
@@ -351,6 +351,9 @@ SSL_callback_ctrl(ssl,SSL_CTRL_SET_TLSEXT_DEBUG_CB,(void (*)(void))cb)
 # define SSL_set_tlsext_debug_arg(ssl, arg) \
 SSL_ctrl(ssl,SSL_CTRL_SET_TLSEXT_DEBUG_ARG,0, (void *)arg)
 
+# define SSL_get_tlsext_status_type(ssl, arg) \
```

Comment:
Well we have lots of getters throughout the library that give the value in the return type (probably more that do that than don't). The whole SSL_ctrl thing does add a layer of confusion which is probably not necessary. I'd kind of like to rip out SSL_ctrl completely. I'm not sure it adds much value. But certainly not for 1.1.0.